exp-models/addon/authenticators/jam-osf-jwt.js

```diff
@@ -23,32 +20,27 @@ export default Base.extend({
                     access_token: accessToken
                 }
             }})
-        }).then(function(res) {
+        }).error(function(/*xhr, status, error*/) {
+            this.transitionToRoute('login');
```

Comment:
Is this transition necessary? It seems to me that experimenter always used to redirect to the login page when needed- was that an automatic behavior of ember simple auth? I'm hesitant to mix routing behavior into the generic logic module unless absolutely necessary; this couples the generic component to a specific application.